src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  border-image: linear-gradient rgba(255, 255, 255, 1);
+  border-image-slice: 1;
+
+  &__top {
+    margin: 32px 19px 0;
+    height: 134px;
+
+    &-image {
+      height: 134px;
+      width: 160px;
+    }
+  }
+
+  &__content {
+    margin-top: 40px;
+    padding: 0 16px;
+
+    &-name {
+      margin: 0;
+      line-height: 18px;
+      font-weight: 500;
+      font-size: 12px;
+      color: $main-color;
+    }
+
+    &-code {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: rgba(97, 96, 112, 1);
+      margin-top: 4px;
+      margin-bottom: 0;
+    }
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    align-self: center;
+    text-align: center;
+    justify-content: space-between;
+    margin-top: 16px;
+    padding: 0 16px;
+
+    &-reviews {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      align-items: right;
+      color: $main-color;
+      margin: 0;
+      padding-top: 1px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 16px;
+    margin-top: 24px;
+
+    &-word {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: rgba(97, 96, 112, 1);
+      margin: 0;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      text-align: right;
+      color: $main-color;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid $second-color;
+    border-radius: 5px;
+    background-color: $second-color;
+    text-align: center;
+    height: 40px;
+    width: 166px;
```

```suggestion
    width: 100%
```
The button should be 100% width of its container